OpenRA.Mods.Common/Traits/ProductionParadrop.cs

```diff
@@ -0,0 +1,170 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Drawing;
+using System.Linq;
+using OpenRA.Activities;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common
+{
+
+	[Desc("Deliver the unit in production via paradrop.")]
+	public class ProductionParadropInfo : ProductionInfo
+	{
+		public readonly string ReadyAudio = "ReinforcementsArrived";
+		[Desc("Cargo aircraft used.")]
+		[ActorReference] public readonly string ActorType = "c17";
```

Comment:
It should be obvious, but I don't like to leave 'obvious' things lying around.
Since this actor must fulfill the `Fly` activity please note in a description that the actor must have the `Aircraft` trait.

Source:
```csharp
		public Fly(Actor self, Target t)
		{
			plane = self.Trait<Aircraft>();
			target = t;
		}
```